corehq/apps/dashboard/models.py

```diff
@@ -329,17 +329,14 @@ def _get_app_url(app):
             )
 
         def _get_view_app_url(app):
-            return reverse('view_app', args=[self.request.domain, app['id']])
+            return reverse('view_app', args=[self.request.domain, app.get_id])
```

Comment:
(random non-blocking thought - it would be nice if we were consistent about `get_id` versus `_id`. i prefer the latter just cause it's more intuitive with how the rest of `couchdbkit` works even though you get syntax warnings in pycharm)